Documentation/user-guides/getting-started.md

```diff
@@ -0,0 +1,156 @@
+# Prometheus Operator
+
+Operators were introduced by CoreOS as a class of software that operates other software, putting operational knowledge collected by humans into software. Read more in the [original blog post](https://coreos.com/blog/introducing-operators.html).
+
+The mission of the Prometheus Operator is to make running Prometheus on top of Kubernetes as easy as possible, while preserving configurability as well as making the configuration Kubernetes native.
+
+To follow this getting started you will need a Kubernetes cluster you have access to. Let's give the Prometheus Operator a spin:
+
+[embedmd]:# (../../deployment.yaml)
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus-operator
+  labels:
+    operator: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        operator: prometheus
+    spec:
+      containers:
+       - name: prometheus-operator
+         image: quay.io/coreos/prometheus-operator:v0.6.0
+         resources:
+           requests:
+             cpu: 100m
+             memory: 50Mi
+           limits:
+             cpu: 200m
+             memory: 100Mi
+```
+
```

Comment:
Can you add the kubectl command to start this? (mostly for consistency with below)